src/main/java/racingcar/controller/WebController.java

```diff
@@ -0,0 +1,51 @@
+package racingcar.controller;
+
+import java.util.List;
+
+import javax.validation.Valid;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import racingcar.dto.RacingCarRequestDto;
+import racingcar.dto.RacingCarResponseDto;
+import racingcar.model.Cars;
+import racingcar.model.Trial;
+import racingcar.service.RacingCarService;
+import racingcar.ui.WebInputConvertor;
+
+@RestController
+public class WebController {
+    private final RacingCarService racingcarService;
+
+    public WebController(RacingCarService racingcarService) {
+        this.racingcarService = racingcarService;
+    }
+
+    @PostMapping("/plays")
+    public ResponseEntity<RacingCarResponseDto> plays(@Valid @RequestBody RacingCarRequestDto request) {
+        List<String> carNames = WebInputConvertor.carNames(request.getNames());
+        Cars cars = Cars.from(carNames);
+        int tryCount = WebInputConvertor.tryCount(request.getCount());
+        Trial trial = new Trial(tryCount);
+        return ResponseEntity.ok()
+            .body(racingcarService.play(cars, trial));
```

Comment:
개인적인 의견입니다 😊

`racingcarService.play(cars, trial)` 이 부분을 따로 선언하여 넘기면 어떨까요 ? 

```suggestion
        Response response = racingcarService.play(cars, trial);
        return ResponseEntity.ok()
            .body(response);
```

이유는, 해당 메서드의 가장 핵심인 부분인 로직이 한눈에 드러나지 않기 때문이에요.

Reply:
추천해본 방식으로 수정했는데, 해당 메서드가 리팩토링 후 간결해져서 구조가 더 잘 드러나는 지는 모르겠어요...👉👈